imports/ui/components/settings/Settings.jsx

```diff
@@ -273,19 +273,36 @@ class SettingsPage extends Component {
                             />
                         {Meteor.user().username ?
 
-                        <Input type='text' label='Username'
-                               name= 'username'
+                            (<span><Input type='text' label='Username'
```

Comment:
@Furqankhanzada sir because we have to show both fields in the form but at a time one field should have a "required" attribute and the other will be optional.